### ISSUE TITLE
Devise::Controllers::Helpers

### ISSUE THREAD

**Issue by danielgrippi** [MikeSofaer/vanna#1]
Including `Devise::Controllers::Helpers` in ApplicationController.rb resolves the Devise-specific `before_filter :authenticate_user!`, but I'm still not able to use the current_user method in my controllers.  This is what happens when I try calling it:

```
undefined local variable or method `current_user' for #<#<Class:0x00000100f44628>:0x00000100f3d648>
```

It seems as if the `define_helpers` method isn't getting triggered in Devise.  Any ideas?

**Comment by danielgrippi** [comment]
Yep, looking at the `define_helpers(mapping)` a little further, it looks like the helper_methods only get added when action_controller gets loaded: https://github.com/plataformatec/devise/blob/master/lib/devise/controllers/helpers.rb#L56

**Comment by danielgrippi** [comment]
my quick and dirty fix for now -- placing this method in application_helper:

```
def current_user
   @current_user ||= warden.authenticate(:scope => :user)
end
```

**Comment by MikeSofaer** [comment]
Weird.  I poked around at this locally, and didn't get the problem, Devise works fine for me with just the include.

That's a weird error message to get, though, I'd expect to see the class name.  Are you calling it in an instance method of a controller that inherits from ApplicationController?

**Comment by rsofaer** [comment]
Can you call define_helpers on the controller class?  Then just call helper_method yourself to put it in the views.
